src/extra/AutoUI/Filters/PersistentFilters.tsx

```diff
@@ -125,10 +123,8 @@ export const PersistentFilters = ({
 	const locationSearch = history?.location?.search ?? '';
 	const storedFilters = React.useMemo(() => {
 		const urlRules = loadRulesFromUrl(locationSearch, schema);
-		return !!urlRules?.length
-			? urlRules
-			: getFromLocalStorage<JSONSchema[]>(filtersRestorationKey) ?? [];
-	}, [locationSearch, schema, filtersRestorationKey]);
+		return !!urlRules?.length ? urlRules : [];
```

Comment:
Is this just v ?
```suggestion
		return urlRules ?? [];
```
After further checking, it seems that `loadRulesFromUrl` always returns an array, so we can just return the result of `loadRulesFromUrl()` directly :)

Reply:
Nice catch :)